definitions/npm/underscore_v1.x.x/flow_v0.13.x-/underscore_v1.x.x.js

```diff
@@ -13,39 +17,82 @@ declare module "underscore" {
 
   declare function flatten<S>(a: S[][]): S[];
 
-  declare function any<T>(list: Array<T>, pred: (el: T)=>boolean): boolean;
-
   declare function each<T>(o: {[key:string]: T}, iteratee: (val: T, key: string)=>void): void;
   declare function each<T>(a: T[], iteratee: (val: T, key: string)=>void): void;
 
-  declare function map<T, U>(a: T[], iteratee: (val: T, n?: number)=>U): U[];
-  declare function map<K, T, U>(a: {[key:K]: T}, iteratee: (val: T, k?: K)=>U): U[];
+  declare function map<T, U>(a: T[], iteratee: (val: T, n: number)=>U): U[];
+  declare function map<K, T, U>(a: {[key:K]: T}, iteratee: (val: T, k: K)=>U): U[];
+  declare function pluck(a: Array<any>, propertyName: string): Array <any>;
+
+  declare function reduce<T, MemoT>(a: Array<T>, iterator: (m: MemoT, o: T)=>MemoT, initialMemo?: MemoT): MemoT;
+  declare function inject<T, MemoT>(a: Array<T>, iterator: (m: MemoT, o: T)=>MemoT, initialMemo?: MemoT): MemoT;
+  declare function foldl<T, MemoT>(a: Array<T>, iterator: (m: MemoT, o: T)=>MemoT, initialMemo?: MemoT): MemoT;
+  declare function reduceRight<T, MemoT>(a: Array<T>, iterator: (m: MemoT, o: T)=>MemoT, initialMemo?: MemoT): MemoT;
+  declare function foldRight<T, MemoT>(a: Array<T>, iterator: (m: MemoT, o: T)=>MemoT, initialMemo?: MemoT): MemoT;
 
   declare function object<T>(a: Array<[string, T]>): {[key:string]: T};
   declare function pairs<T>(o: {[key:string]: T}): Array<[string, T]>;
 
   declare function every<T>(a: Array<T>, pred: (val: T)=>boolean): boolean;
+  declare function all<T>(a: Array<T>, pred: (val: T)=>boolean): boolean;
+  declare function some<T>(a: Array<T>, pred: (val: T)=>boolean): boolean;
+  declare function any<T>(a: Array<T>, pred: (val: T)=>boolean): boolean;
+
+  declare function intersection<T>(...arrays: Array<Array<T>>): Array<T>;
+  declare function difference<T>(array: Array<T>, ...others: Array<Array<T>>): Array<T>;
 
   declare function initial<T>(a: Array<T>, n?: number): Array<T>;
   declare function rest<T>(a: Array<T>, index?: number): Array<T>;
 
+  declare function first<T>(a: Array<T>): T;
```

Comment:
Swap the declare order. The one with more parameters first.
You can include the array check afterwards.